website/app/controllers/show.js

```diff
@@ -8,6 +8,21 @@ import { inject as service } from '@ember/service';
 
 import { showdownConfig } from '../shared/showdown-config';
 
+const getTOCs = (container) => {
+  let headings = [];
+  container.querySelectorAll(`:scope > h2, :scope > h3`).forEach((element) => {
+    // we need to add a class to avoid the element being hidden behind the fixed top header
+    element.classList.add('doc-page-sidecar-scroll-margin-top');
```

Comment:
it's not clear in the preview app what this is doing as there is already `margin-top` being applied to these same elements. something I'm missing?

Reply:
they're not the same thing. one is the actual margin, the other one is how much space to leave before the element when it's scrolled into the viewport